tests/modules/images/dominant-color/dominant-color-test.php

```diff
@@ -108,6 +108,25 @@ public function test_tag_add_adjust_to_image_attributes( $image_path, $expected_
 		$this->assertEquals( $filtered_image_mock_lazy_load, $filtered_image_tags_not_added );
 	}
 
+	/**
+	 * Tests dominant_color_img_tag_add_dominant_color() do not override the style attribute.
+	 *
+	 * @covers ::dominant_color_img_tag_add_dominant_color
+	 */
+	public function test_dominant_color_do_not_override_the_style_attribute_in_img_tag() {
```

Comment:
```suggestion
	public function test_dominant_color_should_not_replace_existing_inline_styles() {
```

Also suggest that the test method description is changed to:

> Tests that dominant_color_img_tag_add_dominant_color() does not replace existing inline styles.

Reply:
Thanks for the review.